<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ODF_Metadata_Form</class>
 <widget class="QWidget" name="ODF_Metadata_Form">
  <property name="windowModality">
   <enum>Qt::WindowModality::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>822</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>1</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="missionTemplateSelectorComboBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>20</y>
     <width>311</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="okPushButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>730</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="missionTemplateSelectorLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>231</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>MISSION TEMPLATE SELECTOR</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="cruiseHeaderGoupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>491</width>
     <height>282</height>
    </rect>
   </property>
   <property name="title">
    <string>CRUISE_HEADER</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="4" colspan="3">
     <widget class="QLineEdit" name="cruiseNumberLineEdit">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QLineEdit" name="countryInstituteCodeLineEdit">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="6" column="2" colspan="5">
     <widget class="QLineEdit" name="platformLineEdit">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="5">
     <widget class="QLabel" name="cruiseDesciptionLabel">
      <property name="text">
       <string>CRUISE_DESCRIPTION:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1" colspan="6">
     <widget class="QLineEdit" name="endDateLineEdit">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="6">
     <widget class="QLabel" name="countryInstituteCodeLabel">
      <property name="text">
       <string>COUNTRY_INSTITUTE_CODE:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="4">
     <widget class="QLabel" name="chiefScientistLabel">
      <property name="text">
       <string>CHIEF_SCIENTIST:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="4">
     <widget class="QLabel" name="organizationLabel">
      <property name="text">
       <string>ORGANIZATION:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="platformLabel">
      <property name="text">
       <string>PLATFORM:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4" colspan="3">
     <widget class="QLineEdit" name="organizationLineEdit">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="4">
     <widget class="QLabel" name="cruiseNumberLabel">
      <property name="text">
       <string>CRUISE_NUMBER:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QLabel" name="startDateLabel">
      <property name="text">
       <string>START_DATE:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2" colspan="5">
     <widget class="QLineEdit" name="startDateLineEdit">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="8" column="5" colspan="2">
     <widget class="QLineEdit" name="cruiseDescriptionLineEdit">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="3">
     <widget class="QLabel" name="cruiseNameLabel">
      <property name="text">
       <string>CRUISE_NAME:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4" colspan="3">
     <widget class="QLineEdit" name="chiefScientistLineEdit">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="endDateLabel">
      <property name="text">
       <string>END_DATE:</string>
      </property>
     </widget>
    </item>
    <item row="7" column="3" colspan="4">
     <widget class="QLineEdit" name="cruiseNameLineEdit">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="cancelPushButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>730</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>CANCEL</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="eventHeaderGroupBox">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>110</y>
     <width>521</width>
     <height>590</height>
    </rect>
   </property>
   <property name="title">
    <string>EVENT_HEADER</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="12" column="0">
     <widget class="QLabel" name="minDepthLabel">
      <property name="text">
       <string>MIN_DEPTH:</string>
      </property>
     </widget>
    </item>
    <item row="17" column="0" colspan="5">
     <widget class="QLabel" name="stationNameLabel">
      <property name="text">
       <string>STATION_NAME:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="6">
     <widget class="QLabel" name="eventNumberLabel">
      <property name="text">
       <string>EVENT_NUMBER:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="9">
     <widget class="QLabel" name="eventQualifier2Label">
      <property name="text">
       <string>EVENT_QUALIFIER2:</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0" colspan="4">
     <widget class="QLabel" name="endLatitudeLabel">
      <property name="text">
       <string>END_LATITUDE:</string>
      </property>
     </widget>
    </item>
    <item row="15" column="0">
     <widget class="QLabel" name="soundingLabel">
      <property name="text">
       <string>SOUNDING:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="12">
     <widget class="QLabel" name="origCreationDateLabel">
      <property name="text">
       <string>ORIG_CREATION_DATE:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="dataTypeLabel">
      <property name="text">
       <string>DATA_TYPE:</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0" colspan="10">
     <widget class="QLabel" name="initialLongitudeLabel">
      <property name="text">
       <string>INITIAL_LONGITUDE:</string>
      </property>
     </widget>
    </item>
    <item row="7" column="7" colspan="6">
     <widget class="QLineEdit" name="endDateTimeLineEdit">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="19" column="0" colspan="10">
     <widget class="QLabel" name="eventCommentsLabel">
      <property name="text">
       <string>EVENT_COMMENTS:</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="7">
     <widget class="QLabel" name="initialLatitudeLabel">
      <property name="text">
       <string>INITIAL_LATITUDE:</string>
      </property>
     </widget>
    </item>
    <item row="8" column="8" colspan="5">
     <widget class="QLineEdit" name="initialLatitudeLineEdit">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="18" column="0" colspan="3">
     <widget class="QLabel" name="setNumberLabel">
      <property name="text">
       <string>SET_NUMBER:</string>
      </property>
     </widget>
    </item>
    <item row="19" column="10" colspan="3">
     <widget class="QLineEdit" name="eventCommentsLineEdit">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="9">
     <widget class="QLabel" name="eventQualifier1Label">
      <property name="text">
       <string>EVENT_QUALIFIER1:</string>
      </property>
     </widget>
    </item>
    <item row="13" column="0" colspan="2">
     <widget class="QLabel" name="maxDepthLabel">
      <property name="text">
       <string>MAX_DEPTH:</string>
      </property>
     </widget>
    </item>
    <item row="14" column="12">
     <widget class="QLineEdit" name="samplingIntervalLineEdit">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="11" column="0" colspan="6">
     <widget class="QLabel" name="endLongitudeLabel">
      <property name="text">
       <string>END_LONGITUDE:</string>
      </property>
     </widget>
    </item>
    <item row="18" column="4" colspan="9">
     <widget class="QLineEdit" name="setNumberLineEdit">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="10" column="5" colspan="8">
     <widget class="QLineEdit" name="endLatitudeLineEdit">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="3" column="10" colspan="3">
     <widget class="QLineEdit" name="eventQualifier2LineEdit">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="5" column="12">
     <widget class="QLineEdit" name="origCreationDateLineEdit">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="8">
     <widget class="QLabel" name="startDateTimeLabel">
      <property name="text">
       <string>START_DATE_TIME:</string>
      </property>
     </widget>
    </item>
    <item row="11" column="8" colspan="5">
     <widget class="QLineEdit" name="endLongitudeLineEdit">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="12" column="3" colspan="10">
     <widget class="QLineEdit" name="minDepthLineEdit">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="6" column="9" colspan="4">
     <widget class="QLineEdit" name="startDateTimeLineEdit">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="13" column="3" colspan="10">
     <widget class="QLineEdit" name="maxDepthLineEdit">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="16" column="0" colspan="11">
     <widget class="QLabel" name="depthOffBottomLabel">
      <property name="text">
       <string>DEPTH_OFF_BOTTOM:</string>
      </property>
     </widget>
    </item>
    <item row="15" column="2" colspan="11">
     <widget class="QLineEdit" name="soundingLineEdit">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="1" column="8" colspan="5">
     <widget class="QLineEdit" name="eventNumberLineEdit">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="4" column="8" colspan="5">
     <widget class="QLineEdit" name="creationDateLineEdit">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="6">
     <widget class="QLabel" name="endDateTimeLabel">
      <property name="text">
       <string>END_DATE_TIME:</string>
      </property>
     </widget>
    </item>
    <item row="17" column="6" colspan="7">
     <widget class="QLineEdit" name="stationNameLineEdit">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="6">
     <widget class="QLabel" name="creationDateLabel">
      <property name="text">
       <string>CREATION_DATE:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="12">
     <widget class="QLineEdit" name="dataTypeLineEdit">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="9" column="11" colspan="2">
     <widget class="QLineEdit" name="initialLongitudeLineEdit">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="14" column="0" colspan="12">
     <widget class="QLabel" name="samplingIntervalLabel">
      <property name="text">
       <string>SAMPLING_INTERVAL:</string>
      </property>
     </widget>
    </item>
    <item row="16" column="12">
     <widget class="QLineEdit" name="depthOffBottomLineEdit">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="2" column="10" colspan="3">
     <widget class="QLineEdit" name="eventQualifier1LineEdit">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="yearLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>341</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>PLEASE ENTER YEAR DATA WAS COLLECTED:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="yearLineEdit">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>60</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
